evaluator/evaluator/app/models.py

```diff
@@ -133,6 +133,24 @@ class MetricResult(MongoModel):
     metrics: dict = Field(..., description="Dictionary of all Metric objects")
 
 
+class EvaluationResult(BaseModel):
+    evaluation_id: str = Field(..., description="Skill Id + metric name")
```

Comment:
Muss jetzt nicht zwingend geändert werden. Aber hat das nen Grund, dass das nicht einfach die ID von der evaluation aus der DB ist?

Reply:
Ja hast Recht. War mir bis jetzt nicht aufgefallen, weil ich nicht so viele Evaluations in meiner db habe. Ich setzte die evaluation id rein.